code/modules/unit_tests/mouse_bite_cable.dm

```diff
@@ -0,0 +1,20 @@
+/// Unit Test to ensure that a mouse bites a cable, gets shocked, and dies.
+/datum/unit_test/mouse_bite_cable
+
+/datum/unit_test/mouse_bite_cable/Run()
+	var/mob/living/simple_animal/mouse/biter = allocate(/mob/living/simple_animal/mouse/cable_lover) // use special subtype that will bypass the probability check to bite on a cable
+	var/obj/structure/cable/wire = allocate(/obj/structure/cable)
+	biter.chew_probability = 100 // Make sure the mouse bites the cable!
+	wire.powernet = new /datum/powernet() // Make sure the cable has a powernet.
```

Comment:
cables create powernets by themselves I believe, this shouldn't be necessary

Reply:
i think i was just being too zealous, i'll look into this more later today

Reply:
![image](https://user-images.githubusercontent.com/34697715/195734517-95776255-18a2-4ef0-82d8-2abebdb41013.png)

You get runtimes if you don't give it a powernet, because it's null:
![image](https://user-images.githubusercontent.com/34697715/195736081-67ef3637-7092-4420-85c5-ed946b5a5086.png)

Why is this? I don't fucking know. I stepped through every single part of the cable's initialize and nothing in Initialize or any proc called through Initialize actually gives it a powernet, maybe that's done later with SSmachines? There's a lot of shit about merging powernets and giving it a powernet if you click on a turf with a cable coil, but spawning in a cable does not give it a powernet in current code.

FUCK

Reply:
maybe it's because the unit tests room doesn't have an APC? idfk

Reply:
cables are fucked upbruh